packages/next/src/server/next-typescript.ts

```diff
@@ -803,6 +805,28 @@ export function createTSPlugin(modules: {
                       })
                     }
                   }
+                } else if (name.text === LEGACY_CONFIG_EXPORT) {
+                  // export const config = { ... }
+                  // Error if using `amp: ...`
+                  const value = declarartion.initializer
```

Comment:
nit: declaration typo (also happening in other places). Not sure if is intentional 
```suggestion
                  const value = declaration.initializer
```

Reply:
Oh good point, thanks! I'll fix all them together.